web/profiles/custom/yalesites_profile/modules/custom/ys_views_basic/assets/js/views-basic.js

```diff
@@ -0,0 +1,39 @@
+((Drupal) => {
+  Drupal.behaviors.ysLevers = {
```

Comment:
Rename this to `Drupal.behaviors.ysViewsBasic` as there already is a `Drupal.behaviors.ysLevers` in the `ys_themes` module. I don't think it would cause a conflict, but just to be safe.